### PR TITLE
Fix for cargo-ligen operation

### PR DESCRIPTION
**Pull request by arrudagates**

### CODE REVIEW

File: core/src/generator/project.rs [hunk 1]
```diff
@@ -14,16 +13,13 @@ pub struct ProjectGenerator {}
...
+        let target_dir = &context.arguments.target_dir;
 
         let generator_version = env!("CARGO_PKG_VERSION");
 
-        // FIXME: This is broken when manifest is a workspace.
-        let package = manifest.package.expect("Couldn't get package.");
-        let name = package.name;
+        let name = Arguments::from_env()
+            .expect("Couldn't get CARGO_LIGEN_ARGUMENTS")
+            .crate_name;
```
Comment (notdanilo) [inline core/src/generator/project.rs:22]: Why are we creating a new instance of Arguments if it's already present in Context?

**Review by notdanilo — COMMENTED** [review]
Just one small thing

File: core/src/generator/project.rs (hunk 1 above)
Comment (arrudagates) [reply core/src/generator/project.rs]: Fixed

**Review by notdanilo — APPROVED** [review]
Good to be merged

**Pull request merged**